tests/specifications/adobe_fonts_test.py

```diff
@@ -22,3 +21,27 @@ def test_check_name_empty_records():
     test_font['name'].names[3].string = b' '
     status, message = list(check(test_font))[-1]
     assert status == FAIL
+
+
+def test_check_consistent_upm():
+    from fontbakery.specifications.adobe_fonts import (
+        com_adobe_fonts_check_consistent_upm as check)
+
+    base_path = 'data/test/source-sans-pro/OTF/'
+
```

Comment:
Please add a comment here: `# We know that all of these reference files have a unitesPerEm value of <some_value> in the head table.`

Reply:
...so that the 2048 value a few lines below is clearly different.